src/flamenco/runtime/fd_executor.c

```diff
@@ -618,7 +618,7 @@ fd_executor_setup_accessed_accounts_for_txn( fd_exec_txn_ctx_t * txn_ctx ) {
 int
 fd_should_set_exempt_rent_epoch_max( fd_rent_t const *       rent,
                                      fd_borrowed_account_t * rec ) {
-  if( fd_pubkey_is_sysvar_id( rec->pubkey ) ) {
+  if( !memcmp( rec->const_meta->info.owner, fd_sysvar_owner_id.key, sizeof(fd_pubkey_t) ) ) {
```

Comment:
can you add a permalink for this. Sometimes agave uses all sysvars and sometimes they reference a list used by this function